wolfssl/wolfcrypt/settings.h

```diff
@@ -798,14 +798,24 @@ static char *fgets(char *buff, int sz, FILE *fp)
         #define WOLFSSL_AES_DIRECT
     #endif
 
-    #include "fsl_common.h"
+    #ifndef FREESCALE_KSDK_1_3
```

Comment:
I'd prefer the logic #ifdef FREESCALE_KSDK_1_3 then include "fsl_device_registers.h" else "fsl_common.h" as its easier to read.